src/components/Categories/MainCategoryTile.vue

```diff
@@ -68,17 +65,40 @@ export default {
 			type: String,
 			default: 'small'
 		},
+		categoryName: {
+			type: String,
+			default: ''
+		},
+		categoryImage: {
+			type: String,
+			// eslint-disable-next-line max-len
+			default: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQZmwTlZfck9YScpFfUPHQ6HVyJ3hU7SnN9Xg&usqp=CAU'
+		},
+		categoryDescription: {
+			type: String,
+			default: ''
+		},
+		numberLoans: {
+			type: Number,
+			default: 0
+		}
 	},
 	components: {
 	},
 	data() {
 		return {
-			image: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQZmwTlZfck9YScpFfUPHQ6HVyJ3hU7SnN9Xg&usqp=CAU'
 		};
 	},
 };
 </script>
 
 <style lang="scss" scoped>
 @import "settings";
+
+.image-sizing {
```

Comment:
This can replaced with `tw-object-cover` and inline style. We're trying to replace all SCSS blocks over time.

Reply:
Hi Casey, yes, sorry! I was just looking through the code I had so far with Eddie. I probably shouldn't have a made a PR as I wasn't really trying to merge it yet. I'll fix all of these things.

Reply:
No worries at all :). I take a look at all `ui` PRs and add comments/questions (though I don't usually approve PRs for other teams unless they request it directly, since I don't always have the context). Definitely let me know if you ever have questions on any comments I add.